linux/deploy_vm/templates/post_install_scripts/get_os_info.sh

```diff
@@ -8,18 +8,34 @@ echo "System environment variables:"
 env | sort
 
 # Get IPv4 address
+echo "Display network interfaces"
+ip -o -f inet addr show
```

Comment:
This line can be put in the below loop?

Reply:
This is for debugging what is the status of ip address, to see whether the network interface is listed at the moment. No need to loop it.